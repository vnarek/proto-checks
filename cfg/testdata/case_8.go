package main

//wrong
func main() {
	x := new(int)
	y = *(&x) //hier
}
