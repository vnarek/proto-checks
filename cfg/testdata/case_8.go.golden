[_t1 = &y]
[x = *_t1]
